client/my-sites/backup/controller.js

```diff
@@ -93,6 +105,19 @@ export function showUnavailableForMultisites( context, next ) {
 
 /* handles /backup/:site, see `backupMainPath` */
 export function backups( context, next ) {
+	debug( 'controller: backups', context.params );
+
+	// When a user visits `/backup`, we don't want to carry over any filter
+	// selection that could've happened in the Activity Log, otherwise,
+	// the app will render the `SearchResults` component instead of the
+	// `BackupStatus`.
+	const state = context.store.getState();
+	const siteId = getSelectedSiteId( state );
+	context.store.dispatch( {
```

Comment:
Is it possible this could cause a race condition for the filter to appear for a split second before updating?

Reply:
I don't think so, because, by default, Redux’s actions are dispatched synchronously. The only way that could ever happen is if we were dispatching an action that is handled in the `data-layer` module because those actions deal with HTTP requests and that makes them asynchronous.

When I was making this change, I had the same concern you have so I made sure to test my beliefs and that they apply to Calypso, and they do.